stats/influxdb/collector.go

```diff
@@ -200,8 +200,8 @@ func (c *Collector) Format(samples []stats.Sample) ([]string, error) {
 }
 
 // GetRequiredSystemTags returns which sample tags are needed by this collector
-func (c *Collector) GetRequiredSystemTags() lib.TagSet {
-	return lib.TagSet{} // There are no required tags for this collector
+func (c *Collector) GetRequiredSystemTags() stats.SystemTagSet {
+	return stats.SystemTagSet(0) // There are no required tags for this collector
```

Comment:
I think you should be able to just say `return 0` here, Go should be smart enough to deal with it

Reply:
Sure, just want to clarify, because the compiler still does conversion, so generated code is the same